src/integration/car/local/LocalCarIntegration.ts

```diff
@@ -0,0 +1,235 @@
+import { CarCatalog, CarCatalogChargeAlternativeTable, CarCatalogChargeOptionTable, CarCatalogConverter } from '../../../types/Car';
```

Comment:
Not the right place, Car integration class must implement the interface which is not the case here.
Move this file to the newly created directory 'bootstrap' where I put the CS Template import.
Proposal for the naming: LocalCarCatalogBootstrap.uploadLocalCarCatalogsFromFile()

Reply:
Done